src/main/java/com/unstoppabledomains/resolution/naming/service/CNS.java

```diff
@@ -82,10 +92,33 @@ public  String getOwner(String domain) throws NamingServiceException {
     }
   }
 
+  @Override
+  public List<DnsRecord> getDns(String domain, List<DnsRecordsType> types) throws NamingServiceException, DnsException {
+    DnsUtils util = new DnsUtils();
+    String[] keys = constructDnsRecords(types);
+    ProxyData data = resolveKeys(keys, domain);
+    List<String> values = data.getValues();
+    Map<String, String> rawData = new HashMap<String, String>();
```

Comment:
```suggestion
    Map<String, String> rawData = new HashMap<>();
```
Right-side typing is redundant since you defined it on a left-side.